streamflow/README.md

```diff
@@ -1 +1,37 @@
+# pystreamflow
+
 pystreamflow enables efficient simulation and inference for rainfall runoff differential equation models of the sort that arise in hydrological modelling of river basins.
+
+## Installation
+
+1. Navigate to `differential-equations-inference-db/streamflow/` (this directory)
+1. For a simple install, run `pip install .`
+1. To install with support for the CVODE solver via scikits.odes, run `pip install .[cvode]`
```

Comment:
So, I tried to install without doing this step but I ran into the following error:

RuntimeError: scikit solver could not be imported

Is scikit.odes required? Or does it just make things faster?

Reply:
Hmm, when exactly did this error appear? Was it when running the notebook? Or did it appear during pip install?

scikit.odes is optional and just makes solving faster. However, if you try to instantiate the model object with the non-default optional argument `solver='scikit'`, it will raise this error if scikits.odes is not available. So if this happened when running the notebook cell which compares the two different solvers, I would consider that the expected behavior. 

Another option would be to raise a warning and revert to the scipy solver in this case.